packages/grafana-ui/src/components/PageLayout/PageToolbar.tsx

```diff
@@ -146,7 +146,7 @@ const getStyles = (theme: GrafanaTheme2) => {
     border-radius: 2px;
 
     @media ${styleMixins.mediaUp(theme.v1.breakpoints.xl)} {
-      max-width: unset;
+      max-width: 60vw;
```

Comment:
rather than setting a seemingly arbitrary max-length, what do we instead think about the title being in a flex container that takes up all remaining width, and truncates?